lib/dams/dams_object.rb

```diff
@@ -597,6 +597,22 @@ def to_solr (solr_doc = {})
 	      insertFacets solr_doc, "subject_topic", load_series(component.series)
 	      insertFacets solr_doc, "subject_topic", load_cruises(component.cruise)
 	      	      	
+          # uncomment for subject anatomy
+          #insertFacets solr_doc, "subject_anatomy", load_anatomys(component.anatomy)
+          #insertFacets solr_doc, "component_#{cid}_subject_anatomy", load_anatomys(component.anatomy)
+          insertFacets solr_doc, "subject_common_name", load_commonNames(component.commonName)
+          insertFacets solr_doc, "component_#{cid}_subject_common_name", load_commonNames(component.commonName)
```

Comment:
I am wondering why we need it either but just make it consistent with subject_topic https://github.com/ucsdlib/damspas/blob/develop/lib/dams/dams_object.rb#L582 in case we use it in the future. Do you want me to remove them all?